File: TheLoversApi/src/main/java/com/strilicherk/theloversapi/feature_auth/login/domain/dto/LoginResponseDTO.java
package com.strilicherk.theloversapi.feature_auth.login.domain.dto;

import com.strilicherk.theloversapi.feature_user.domain.dto.UserResponseDTO;

public record LoginResponseDTO(
        String refreshToken,
        String jwtToken,
        Boolean existingUser,
        UserResponseDTO userResponseDTO
) {
}

File: TheLoversMobileWeb/composeApp/src/commonMain/kotlin/org/example/thelovers/feature_auth/data/LoginResponseDTO.kt
package org.example.thelovers.feature_auth.data

import kotlinx.serialization.Serializable
import org.example.thelovers.feature_user.data.dto.UserResponseDTO

@Serializable
data class LoginResponseDTO(
    val jwtToken: String,
    val refreshToken: String,
    val user: UserResponseDTO
)
